lib/workers/repository/finalise/repository-statistics.spec.ts

```diff
@@ -120,27 +119,24 @@ describe('workers/repository/finalise/repository-statistics', () => {
       getCacheSpy.mockReturnValueOnce(cache);
       isCacheModifiedSpy.mockReturnValueOnce(false);
       runBranchSummary();
-      expect(logger.debug).toHaveBeenCalledWith(
-        {
-          cacheModified: false,
-          baseBranches: [
-            {
-              branchName: 'main',
-              sha,
-            },
-            {
-              branchName: 'dev',
-              sha,
-            },
-          ],
-          branches: [
-            { ...expectedMeta, branchName: 'b1' },
-            { ...expectedMeta, branchName: 'b2' },
-          ],
-          inactiveBranches: ['b3'],
-        },
-        `Branch summary`
-      );
+      expect(logger.debug).toHaveBeenCalledWith({
+        cacheModified: false,
+        baseBranches: [
+          {
+            branchName: 'main',
+            sha,
+          },
+          {
+            branchName: 'dev',
+            sha,
+          },
+        ],
+        branches: [
+          { ...expectedMeta, branchName: 'b1' },
+          { ...expectedMeta, branchName: 'b2' },
+        ],
+        inactiveBranches: ['b3'],
+      });
```

Comment:
```suggestion
        ],
          inactiveBranches: ['b3'],
        },
        `Branch summary`
      );
```

Reply:
Didnt work. It seems the cache is not getting initialized in the second test. Will need some debugging.